docs/blog/contributor-development-maintenance-guide.en-US.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Contributor development maintenance guide
+date: 2023-03-10
+author: kiner-tang
+---
+
+Hi, my name is [kiner-tang](https://github.com/kiner-tang). My personal work is closely related to Ant Design, so I can be considered a heavy user of Ant Design.It is for this reason that I am slowly working my way up from a user to a Contributor, contributing some of the problems I encountered and the new features I concluded to the community, and finally being honored to become a member of Ant Design Collaborator.
```

Comment:
Hi, I'm [kiner-tang](https://github.com/kiner-tang) who is a heavy user of Ant Design. As the same reason, I come up to be a contributor for fixing bug and some features. Finally, it's my honer to become a member of Ant Design Collaborator.

Reply:
done